accessibility_monitoring_platform/apps/reports/models.py

```diff
@@ -203,3 +203,15 @@ class Meta:
 
     def __str__(self) -> str:
         return str(f"{self.section}: Table row {self.row_number}")
+
+
+class ReportVisitsMetrics(models.Model):
```

Comment:
Django admin has not been enabled for this new model.

Reply:
Added